add faq (#9292)
open-mmlab/mmdetection: docs/en/notes/faq.md

  1. Uninstall existing `mmcv-lite` in the environment using `pip uninstall mmcv-lite`.
  2. Install `mmcv` following the [installation instruction](https://mmcv.readthedocs.io/en/2.x/get_started/installation.html).

> - "Microsoft Visual C++ 14.0 or graeter is required" during installation on Windows.

Reviewer: Will this happen in installing pycocotools?

Author: yes, `building 'pycocotools._mask' extension`.

Reviewer 2: Then can we indicate this in the FAQ so that readers could better understand the cause?

Author: > Then can we indicate this in the FAQ so that readers could better understand the cause?

OK, I have added it.

>   The reason is that the corresponding C ++ environment compilation dependencies are lacking, and you need to download it at Microsoft officials [visual-cpp-build-tools](https://visualstudio.microsoft.com/zh-hans/visual-cpp-build-tools/),

Reviewer 2: The reason is that there lacks corresponding C ++ compilation dependencies in the environment and you need to

  1. Uninstall existing `mmcv-lite` in the environment using `pip uninstall mmcv-lite`.
  2. Install `mmcv` following the [installation instruction](https://mmcv.readthedocs.io/en/2.x/get_started/installation.html).

- "Microsoft Visual C++ 14.0 or graeter is required" during installation on Windows.

>   This error will happen in installing pycocotools, something like `building 'pycocotools._mask' extension`.
>   The reason is that there lacks corresponding C++ compilation dependencies in the environment and you need to download it at Microsoft officials [visual-cpp-build-tools](https://visualstudio.microsoft.com/zh-hans/visual-cpp-build-tools/),
>   Select the "Use C ++ Desktop Development" option to install the minimum dependencies, and then re-install.
> 

Reviewer 2: ```suggestion
  This error happens when building the 'pycocotools._mask' extension of pycocotools and the environment lacks corresponding C++ compilation dependencies. You need to download it at Microsoft officials [visual-cpp-build-tools](https://visualstudio.microsoft.com/zh-hans/visual-cpp-build-tools/),  select the "Use C ++ Desktop Development" option to install the minimum dependencies, and then reinstall pycocotools.

```
